package.json

```diff
@@ -15,41 +15,41 @@
     "node": ">=0.10"
   },
   "dependencies": {
-    "bem-environ": "~1.4.0"
+    "bem-environ": "^1.4.0"
```

Comment:
we may drop quite a few deps if we get rid of bem-tools in a lib but as far as I know some people use it for `bem create` command